src/Command/AsanaCheckBoardIdsCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Command;
+
+use App\Asana\AsanaApiClient;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'app:asana:check-board-ids',
+    description: 'Add a short description for your command',
+)]
+class AsanaCheckBoardIdsCommand extends Command
```

Comment:
Are these not events rather than boards?

Reply:
After reading the README it makes sense to call this `AsanaCheckBoardIdsCommand`.

Perhaps the environment variables, e.g. `ASANA_NEW_EVENT` should contain the word board, but that decision was probably made many years ago.